lib/simulation/openfisca/mapping/index.js

```diff
@@ -121,26 +116,18 @@ var applyRessources = function(mesAidesEntity, openFiscaEntity, mappingSchema, s
 
 var setNonInjectedPrestationsToZero = function(openFiscaEntity, dateDeValeur) {
     var periods = getPeriods(dateDeValeur);
-    var concernedPeriods = ['moisPrecedent', 'moisMMoins2', 'moisMMoins3', 'anneeGlissante'];
+    var concernedPeriods = periods.last12Months.concat([periods.rollingYear]);
 
-    _.forEach(concernedPeriods, function(periodKey) {
-        var period = periods[periodKey].key;
+    _.forEach(concernedPeriods, function(period) {
         var prestations = _.pick(PRESTATIONS, function(definition) {
             return definition.type == Number;
         });
 
         _.forEach(prestations, function(definition, prestationName) {
-            if (! openFiscaEntity[prestationName]) {
-                openFiscaEntity[prestationName] = {};
-            }
-            if (! openFiscaEntity[prestationName][period]) {
-                openFiscaEntity[prestationName][period] = 0;
-            }
+            openFiscaEntity[prestationName] = openFiscaEntity[prestationName] || {};
+            openFiscaEntity[prestationName][period.key] = openFiscaEntity[prestationName][period.key] || {};
```

Comment:
This doesn't seem equivalent to the previous code, which defaulted to `0` rather than `{}`.